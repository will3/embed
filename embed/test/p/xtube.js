const test = require('../utils/test');

test('xtube',
  'https://www.xtube.com/video-watch/ex-asian-girl-554909');
