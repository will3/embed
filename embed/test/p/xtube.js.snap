// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` xtube 1`] = `
Object {
  "originalUrl": "https://www.xtube.com/video-watch/ex-asian-girl-554909",
  "site_name": undefined,
  "title": "ex asian ***** | XTube Porn Video from moonum",
  "video": Object {
    "description": undefined,
    "height": "480",
    "title": undefined,
    "url": "https://www.xtube.com/video-watch/embedded/ex-asian-554909?embedSize=big",
    "width": "640",
  },
}
`;
